redisearch/aggregation.py

```diff
@@ -86,8 +86,6 @@ class Group(object):
     This object automatically created in the `AggregateRequest.group_by()`
     """
     def __init__(self, fields, reducers):
-        if not fields:
```

Comment:
@quintonparker 
Why remove this and then remove the test?
Aren't they both relevant?

Reply:
hey @ashtul 

see https://github.com/RediSearch/redisearch-py/issues/67

Reply:
basically, aggregation queries that is possible on the cli is not reproducible with redisearch-py because this code is enforcing the `fields` var to be non-empty